packages/vkui/src/components/HorizontalCell/CellEllipsisText/EllipsisText.module.css

```
@@ -0,0 +1,10 @@
+.EllipsisText {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  max-inline-size: calc(
+    var(--vkui_internal--side_cell_width, 128px + var(--vkui_internal--side_cell_gap) * 2) -
+      var(--vkui_internal--side_cell_gap) * 2
+  );
```

```suggestion
  max-inline-size: 100%
```

использование `--vkui_internal--side_cell_width` и `128px` делает компонент зависимым от `HorizontalCell`, а у нас задача сделать компонент самодостаточным

`maxWidth` поможет пользователям подогнать ширину под себя если вдруг у родителя не будут ограничение ширины